tests/unit/test-cmd/test.sign.js

```diff
@@ -248,8 +248,11 @@ describe('sign', () => {
             success: false,
           }),
         })
-        .then((result) => {
-          assert.equal(result.success, false);
+        .then(makeSureItFails())
+        .catch((error) => {
+          assert.instanceOf(error, UsageError);
+          assert.match(error.message,
+            /Add on could not be signed, exiting with status 1/);
```

Comment:
I think the error message can just be this:

    The WebExtension could not be signed

A couple reasons:
* the exist status isn't so important
* for consistency, we try to refer to add-ons as WebExtensions